sklearn/utils/tests/test_threaded_libs.py

```diff
@@ -0,0 +1,149 @@
+import os
+
+import pytest
+
+from sklearn.utils.testing import SkipTest
+from sklearn.utils._threaded_libs import get_thread_limits
+from sklearn.utils._threaded_libs import _set_thread_limits
+from sklearn.utils._threaded_libs import get_openblas_version
+from sklearn.utils._threaded_libs import thread_pool_limits
+from sklearn.utils._threaded_libs import _CLibsWrapper
+from sklearn.utils._threaded_libs_helpers import check_num_threads
+
+
+SKIP_OPENBLAS = get_openblas_version() is None
+
+
+def test_openmp_enabled():
+    # Check that an OpenMP library is loaded
+    limits = get_thread_limits()
+
+    assert not all([lib is None for lib in [limits['openmp_llvm'],
```

Comment:
This would create a generator.

```
In [1]: any(True for i in range(2))                                             
Out[1]: <generator object <genexpr> at 0x7fb5eff82308>

In [2]: any([True for i in range(2)])                                           
Out[2]: True
```